hedera-node/hedera-app/src/xtest/java/contract/AbstractContractXTest.java

```diff
@@ -187,6 +191,22 @@ protected Address addressOf(@NonNull final Bytes address) {
         return Address.wrap(Address.toChecksumAddress(new BigInteger(1, address.toByteArray())));
     }
 
+    private void setupFeeManager() {
+        try {
+            var feeSchedule = loadFeeScheduleFromJson("feeSchedules.json");
```

Comment:
This reads the data into the old protobuf implementation. It's probably ok for now, but we must remember to fix this.

Actually, `GenesisSchema` contains functionality to parse `feeSchedules.json`. Maybe we can make it available. @rbair23, any thoughts?

Reply:
I could not find a way to effectively create a Fake.  I got addressed this issue by reading from a binary file as this seems to be the standard method used in xtests.